src/analog/CommandBinaryBuffer.ts

```diff
@@ -1721,6 +1721,15 @@ class CommandBinaryBuffer extends BinaryBuffer {
         this.setUint16(parameter.resistanceStopThreshold, false);
     }
 
+    private getMqttSessionConfig (): IParameterMqttSessionConfig {
```

Comment:
```suggestion
    private getMqttSessionConfigParameter (): IParameterMqttSessionConfig {
```

and 'setMqttSessionConfig' -> 'setMqttSessionConfigParameter' too

Reply:
++